gnome-shell/src/gnome-shell-sass/_common.scss

```diff
@@ -153,15 +153,15 @@ StScrollBar {
   -barlevel-border-width: 1px;
   -slider-handle-radius: 8px;
   -slider-handle-border-width: 1px;
-  -slider-handle-border-color: $borders_color;
+  -slider-handle-border-color: darken($alt_borders_color, 3%);
   color: lighten($bg_color, 4%);
   &:hover { color: $_hover_bg_color; }
   &:active { color: $_active_bg_color; }
 
   /* START: VERSION SPECIFIC CODE (< 3.29.90) */
   -slider-height: 0.2em; //increase the height because of the border removal
   -slider-background-color:  mix($borders_color, $bg_color, 50%);
-  -slider-border-color: $borders_color; //trough border color
+  -slider-border-color: darken($alt_borders_color, 3%); //trough border color
```

Comment:
same as above

Reply:
woopsie! yeah "debugging" via snap and test xD